File: MissionControlApp-SPA/src/app/_models/mission.ts
import { Accelerator } from './accelerator';
import { Platform } from './platform';
import { MissionTeamMember } from './missionteammember';
import { Assessment } from './assessment';
import { UseCase } from './usecase';

export interface Mission {
    missionId: number;
    userId: number;
    userName: string;
    knownAs: string;
    missionName: string;
    industryAlias: string;
    businessFunctionAlias: string;
    missionStatusAlias: string;
    challenge: string;
    desiredOutcome: string;
    businessImpact: string;
    estimatedRoi: number;
    actualRoi: number;
    actualCost: number;
    timeFrame: number;
    dateCreated: Date;
    active: boolean;
    accelerators?: Accelerator[];
    platforms?: Platform[];
    missionTeam?: MissionTeamMember[];
    missionAssessment?: Assessment;
    public: boolean;
    useCases?: UseCase[];
}

File: MissionControlApp.API/Models/Mission.cs
using System;
using System.Collections.Generic;

namespace MissionControlApp.API.Models
{
    public class Mission
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public virtual User User { get; set; }
        public string MissionName { get; set; }
        public int IndustryId { get; set; }
        public virtual Industry Industry { get; set; }
        public int BusinessFunctionId { get; set; }
        public virtual BusinessFunction BusinessFunction { get; set; }
        public string Challenge { get; set; }
        public string DesiredOutcome { get; set; }
        public string BusinessImpact { get; set; }
        public ulong EstimatedRoi { get; set; }
        public ulong ActualRoi { get; set; }
        public ulong ActualCost { get; set; }
        public int TimeFrame { get; set; }
        public virtual ICollection<MissionAccelerator> MissionAccelerators { get; set; }
        public virtual ICollection<MissionPlatform> MissionPlatforms { get; set; }
        public virtual ICollection<MissionTeam> MissionTeam { get; set; }
        public virtual MissionAssessment MissionAssessment { get; set; }
        public bool Public { get; set; }
        public int MissionStatusId { get; set; }
        public virtual MissionStatus MissionStatus { get; set; }
        public DateTime? DateCreated { get; set; }
        public bool Active { get; set; }
        public Mission()
        {
            DateCreated = DateTime.Now;
            Active = false;
        }
    }
}